### ISSUE TITLE
Slow taps reactions in the simulator while debugging

### ISSUE THREAD

**Issue by RedMickey** [microsoft/vscode-react-native#1683]
I'm experiencing this again. Reactions to taps in the simulator (`iOS 15`) are incredibly slow. `VsCode Version: 1.61.0 (Universal)` and `ReactNativeTools v1.7.0`. Debugging by attaching to packager. Here's my launch.json settings:

```json
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Attach to packager",
            "cwd": "${workspaceFolder}",
            "type": "reactnative",
            "request": "attach"
        }
    ]
}
```

Update: I'll also add that this does not happen when debugging in Chrome only in VsCode.

_Originally posted by @lukaspechar in https://github.com/microsoft/vscode-react-native/issues/1517#issuecomment-939814825_

**Comment by gclark-eightfold** [comment]
Thank god. I thought it might have just been my project since we have a fair bit of console statements. I only had this issue after upgrading my extension from `1.6.1` to `1.7.0`, and I had it with both the `Attach to packager` and `Debug in Exponent` default scripts.
Sorry for the dumb question, but when would the fixes in your PR find its way into the production extension?

**Comment by lukaspechar** [comment]
> OMG! That was it!
> Current extension version debbuger made app incredibly laggy and buggy. I'm running on android device (was trying 2 different devices) and macos and it's not possible to use debugger from 1.7.0 version. Once downgraded to 1.6.1 all become fine.

That is a temporary workaround, let's see if the devs can sort it out with a fix :)

**Comment by electroidru** [comment]
> We've released React Native Tools [v1.7.1](https://github.com/microsoft/vscode-react-native/releases/tag/1.7.1) with the fix for this performance issue. Could you please update the extension and try it out?

I did quick test, debugger is working just fine for me.

**Comment by RedMickey** [comment]
Thanks for your assistance! Since it seems the problem is no longer reproducible, I'm closing the issue.

**Comment by lukaspechar** [comment]
> We've released React Native Tools [v1.7.1](https://github.com/microsoft/vscode-react-native/releases/tag/1.7.1) with the fix for this performance issue. Could you please update the extension and try it out?

I can confirm the same. All working nicely. Thanks for sorting this.
